django_project/ta_system/templates/ta_system/home.html

```diff
@@ -1,4 +1,19 @@
 {% extends "ta_system/base.html" %}
 {% block content %}
-    <h1>Home Page</h1>
-{% endblock content %}
\ No newline at end of file
+    <div class="content-section">
+        {% if last_system_status.status %}
+            <h3> The TA Application System is Open. Apply now!</h3>
+        {% elif last_system_status.date_changed %}
+            <h3> The TA Application System is Closed at {{last_system_status.date_changed}}.</h3>
+        {% else %}
+            <h3> The TA Application System is Closed.</h3>
+        {% endif %}
+    </div>
+    <div class="content-section">
+        {% if last_system_status.status %}
+            <h2> Application Form here </h2>
+        {% else %}
+            <h2>Application Form Not Available</h2>
```

Comment:
I don't think we need to say "Application Form Not Available" when the system is closed. Just saying the system is closed is enough.

Reply:
code deleted